include/dlaf/eigensolver/eigensolver/api.h

```diff
@@ -0,0 +1,32 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2022, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+#include "dlaf/common/vector.h"
+#include "dlaf/matrix/matrix.h"
+#include "dlaf/types.h"
+
+namespace dlaf {
+namespace eigensolver {
+
+template <class T, Device device>
+struct ReturnEigensolverType {
+  common::internal::vector<BaseType<T>> eigenvalues;
```

Comment:
Presumably in the real version this can't be part of the return type, right? (Assuming it's supposed to be async.)

Reply:
Yes, it will be replaced by the async return type used in the tridiagonal eigensolver.